### ISSUE TITLE
`reflect.Pointer` is not compatible with go lower than 1.18

### ISSUE THREAD

**Issue by sorcererxw** [cloudwego/frugal#12]
**Describe the bug**

I cannot compile my project with go 1.16. 

Error message:

```
15# github.com/cloudwego/frugal/internal/rt
16../compile_path/pkg/mod/github.com/cloudwego/frugal@v0.1.2/internal/rt/runtime.go:120:14: undefined: reflect.Pointer
```

The `reflect.Pointer` is added to go in 1.18 https://github.com/golang/go/commit/17910ed4ff5a3cb3dcf4367d4af23ad5a7fe5809 . But this package's mod version is 1.15, which has caused some misunderstandings.

Please use `reflect.Ptr` to ensure compatibility with lower versions.

**To Reproduce**

Build this project with a version of go lower than 1.18.

**Comment by gidd-bud** [comment]
+1

**Comment by chenzhuoyu** [comment]
Fixed in [4c936e8](https://github.com/cloudwego/frugal/commit/4c936e80201259b5c005fdc4de7be820312b672c)
